test/test-functions

```diff
@@ -250,12 +250,12 @@ run_nspawn() {
     fi
 
     if [[ "$UNIFIED_CGROUP_HIERARCHY" = "hybrid" ]]; then
```

Comment:
Should you support `$SYSTEMD_NSPAWN_UNIFIED_HIERARCHY` here as well? Maybe only `$SYSTEMD_NSPAWN_UNIFIED_HIERARCHY` if this is internal and we control it?

Right now it looks a bit confusing that we renamed it mostly everywhere, except for here...

Reply:
UNIFIED_CGROUP_HIERARCHY=hybrid|default|unified|legacy is out internal test setting.
SYSTEMD_NSPAWN_UNIFIED_HIERARCHY=0|1 is the nspawn setting.

Reply:
I guess that looks good.

I think I was perceiving this as confusing, but it's actually splitting two separate things into two separate names, so I guess that's good.